frontend/src/layout/Burger.tsx

```diff
@@ -16,28 +16,52 @@ export const BurgerMenu: React.FC<BurgerMenuProps> = ({ hide, items }) => (
             bottom: 0,
             left: 0,
             right: 0,
-            zIndex: 3017,
+            zIndex: 1000,
             backgroundColor: "#000000a0",
         }}
     >
-        <div css={{
+        <div tabIndex={-1} css={{
             position: "absolute",
             display: "flex",
             flexDirection: "column",
             justifyContent: "space-between",
             top: 0,
             right: 0,
-            backgroundColor: "var(--grey97)",
+            backgroundColor: "white",
             height: "100%",
             width: "clamp(260px, 75%, 450px)",
             overflowY: "auto",
             borderTop: "1px solid var(--grey80)",
-            gap: 16,
-            "& > *:first-child": {
-                borderBottom: "1px dashed var(--grey80)",
+            "> :nth-child(2)": { marginTop: 26 },
+            li: {
+                borderRadius: 4,
+                margin: 8,
+                borderBottom: "none",
```

Comment:
Mhhhhh I'm not sure we are guaranteed that this actually has higher specificity than the statement it "resets". Is it easily possible to instead change the `borderBottom: 2px solid white` to only apply outside the burger menu? :thinking:

Reply:
On second thought: I think the code of the nav needs a bit of a refactor and I already have an idea. WIll do so after this PR landed, so we can ignore this for now I think.

Reply:
Hmm you're right, I think the `& > li: borderBottom: 2px solid white` from `LinkList` should in theory have highter specificity. But it appears to get overwritten anyway by the the above. 🧐 Not sure why.  But since it does, and you are planning to refactor it I guess I will indeed keep it like this for now.